test-utils/generators/UserAnswersEntryGenerators.scala

```diff
@@ -24,6 +24,11 @@ import play.api.libs.json.{JsValue, Json}
 
 trait UserAnswersEntryGenerators extends PageGenerators with ModelGenerators {
 
+  implicit lazy val arbitraryOperatePackagingSitesUserAnswersEntry: Arbitrary[(OperatePackagingSitesPage.type, JsValue)] =
+    Arbitrary {
+      for {
+        page  <- arbitrary[OperatePackagingSitesPage.type]
+
```

Comment:
Believe you lost this bit in a rebase from line 31:        
      value <- arbitrary[Boolean].map(Json.toJson(_))
      } yield (page, value)
    }